test_files/import_empty/import_empty.js

```diff
@@ -1,7 +1,9 @@
 /**
- * @fileoverview added by tsickle
- * @suppress {checkTypes} checked by tsc
+ * @fileoverview Make sure tsickle does not crash on empty imports.
+ * @suppress {checkTypes,extraRequire} checked by tsc
```

Comment:
BTW, I have a pending CL to make the suppressions match on all transformation codepaths.